### PR TITLE
Internal: cancel build with only .md changes

### PR DESCRIPTION
**Pull request by hzoo**
got linked to https://github.com/facebook/react/pull/2000 by @RReverser from @taion

### CODE REVIEW

**Comment by hzoo** [comment]
Ok this doesn't seem to be working - actually not on this commit either

File: .travis.yml [hunk 1]
```diff
@@ -5,6 +5,15 @@ language: node_js
 cache:
   directories:
   - node_modules
+before_install:
+- |
+    if [ "$TRAVIS_PULL_REQUEST" != "false" ]; then
```
Comment (jridgewell) [inline .travis.yml]: Did you mean `!=`?

Comment (taion) [inline .travis.yml]: It looks like none of that machinery should be required any more: https://github.com/facebook/react/commit/08b1515f7fb82c64b3f218bf60964b6b75fa6be6

Comment (jridgewell) [inline .travis.yml]: The only time [we](https://github.com/ampproject/amphtml/blob/master/.travis.yml#L21-L25) ran into this is when there's a branch being tested. PRs worked perfectly.

**Comment by danez** [comment]
https://docs.travis-ci.com/user/environment-variables/#Default-Environment-Variables

File: .travis.yml (hunk 1 above)
Comment (hzoo) [inline .travis.yml]: it didn't seem to work without the check but i'l test again https://github.com/babel/babel/pull/4556

**Pull request merged**